Document/0x04i-Testing-User-Privacy-Protection.md

```diff
@@ -0,0 +1,136 @@
+- An app that records a user’s screen and doesn't treat this data as personal or sensitive data that is subject to this policy.
+
+Since we keep talking about location, contacts, screen recordings, etc., you probably have noticed that all of this is closely related to app permissions. App developers must explain to the user why their app needs the permissions it requests. Both [iOS](https://developer.apple.com/design/human-interface-guidelines/ios/app-architecture/requesting-permission/) and [Android](https://developer.android.com/training/permissions/requesting.html#explain) have specific guidelines and best practices for that.
+
+> App Permissions have its own dedicated requirement in the OWASP MASVS, we suggest that you refer to the related test "Testing App Permissions (MSTG-PLATFORM-1)" for [Android](0x05h-Testing-Platform-Interaction.md#testing-app-permissions-mstg-platform-1) and [iOS](0x06h-Testing-Platform-Interaction.md#testing-app-permissions-mstg-platform-1).
+
+### Learn More
+
+You can learn more about this and other privacy related topics here:
+
+- [iOS App Privacy Policy](https://developer.apple.com/documentation/healthkit/protecting_user_privacy#3705073)
+- [iOS Privacy Details Section on the App Store](https://developer.apple.com/app-store/app-privacy-details/)
+- [iOS Privacy Best Practices](https://developer.apple.com/documentation/uikit/protecting_the_user_s_privacy)
+- [Android App Privacy Policy](https://support.google.com/googleplay/android-developer/answer/9859455#privacy_policy)
+- [Android Data Safety Section on Google Play](https://support.google.com/googleplay/android-developer/answer/10787469)
+- [Android Privacy Best Practices](https://developer.android.com/privacy/best-practices)
+
+## Testing User Education (MSTG-STORAGE-12)
+
+### Testing User Education on Data Privacy on the App Marketplace
+
+At this point we're only interested into knowing which privacy related information is being disclosed by the developers and try to evaluate if it seems reasonable (similarly as you'd do when testing for permissions).
+
+> It's possible that the developers are not declaring certain information that is indeed being collected and/or shared, but that's a topic for a different test extending this one here.
+
+### Static Anaylsis
+
+You can follow these steps:
+
+1. Search for the app in the corresponding app marketplace (e.g. Google Play, App Store).
+2. Go to the section ["Privacy Details"](https://developer.apple.com/app-store/app-privacy-details/) (App Store) or ["Safety Section"](https://android-developers.googleblog.com/2021/05/new-safety-section-in-google-play-will.html) (Google Play).
+3. Verify if there's any infomation available at all.
+4. Compare the information available against the actual context of the app. Does everything make sense?
```

Comment:
That seems to be quite a big ask. Should this really be the job a penetration tester to verify? Or should the pentester just record and verify the app inspection and make it part of the deliverable? 
A pentester might not want to give assurance for privacy violations in such a detail as this could become quite time consuming and should focus on vulnerabilities (including raising suspicious app permissions) in the app and endpoint.

Reply:
I'll modify the conclusion/assessment paragraph below to reflect that for this test, verifying this is actually enough. After that I'll add the "app activity monitoring/exported Privacy Report" below as "Dynamic Analysis", being an optional step providing evidence.